diff --git a/build-custom-worker.js b/build-custom-worker.js
index b741ebc639ffc16a56001bb33212a6d4752be9f5..aa12a73fd35e94031495a5ad245bc8d97b71b68b 100644
--- a/build-custom-worker.js
+++ b/build-custom-worker.js
@@ -34,10 +34,30 @@ const buildCustomWorker = ({ id, basedir, destdir, plugins, success, minify }) =
       main: customWorkerEntry
     },
     resolve: {
-      extensions: ['.ts', '.js']
+      extensions: ['.ts', '.js'],
+      fallback: {
+        module: false,
+        dgram: false,
+        dns: false,
+        path: false,
+        fs: false,
+        os: false,
+        crypto: false,
+        stream: false,
+        http2: false,
+        net: false,
+        tls: false,
+        zlib: false,
+        child_process: false
+      }
     },
     module: {
       rules: [
+        {
+          test: /\.(graphql|gql)$/,
+          use: "graphql-tag/loader",
+          exclude: /node_modules/
+        },
         {
           test: /\.(t|j)s$/i,
           use: [
diff --git a/build-fallback-worker.js b/build-fallback-worker.js
index af3ce017de6f008500f567f58f4f96749a27004e..109ac79286a1cc19c3504252b46d60bfe9b7c327 100644
--- a/build-fallback-worker.js
+++ b/build-fallback-worker.js
@@ -68,10 +68,30 @@ const buildFallbackWorker = ({ id, fallbacks, basedir, destdir, success, minify
       main: fallbackJs
     },
     resolve: {
-      extensions: ['.js']
+      extensions: ['.js'],
+      fallback: {
+        module: false,
+        dgram: false,
+        dns: false,
+        path: false,
+        fs: false,
+        os: false,
+        crypto: false,
+        stream: false,
+        http2: false,
+        net: false,
+        tls: false,
+        zlib: false,
+        child_process: false
+      }
     },
     module: {
       rules: [
+        {
+          test: /\.(graphql|gql)$/,
+          use: "graphql-tag/loader",
+          exclude: /node_modules/
+        },
         {
           test: /\.js$/i,
           use: [
