diff --git a/build-custom-worker.js b/build-custom-worker.js
index b741ebc639ffc16a56001bb33212a6d4752be9f5..b55643c34cf15cca9ba0f060f0d7f4a372f9d74e 100644
--- a/build-custom-worker.js
+++ b/build-custom-worker.js
@@ -34,7 +34,22 @@ const buildCustomWorker = ({ id, basedir, destdir, plugins, success, minify }) =
       main: customWorkerEntry
     },
     resolve: {
-      extensions: ['.ts', '.js']
+      extensions: ['.ts', '.js'],
+      fallback: {
+        module: false,
+        dgram: false,
+        dns: false,
+        path: false,
+        fs: false,
+        os: false,
+        crypto: false,
+        stream: false,
+        http2: false,
+        net: false,
+        tls: false,
+        zlib: false,
+        child_process: false
+      }
     },
     module: {
       rules: [
diff --git a/build-fallback-worker.js b/build-fallback-worker.js
index af3ce017de6f008500f567f58f4f96749a27004e..0b6d120fe0216ffc0e2320aeed4b5b9445543a8d 100644
--- a/build-fallback-worker.js
+++ b/build-fallback-worker.js
@@ -68,7 +68,22 @@ const buildFallbackWorker = ({ id, fallbacks, basedir, destdir, success, minify
       main: fallbackJs
     },
     resolve: {
-      extensions: ['.js']
+      extensions: ['.js'],
+      fallback: {
+        module: false,
+        dgram: false,
+        dns: false,
+        path: false,
+        fs: false,
+        os: false,
+        crypto: false,
+        stream: false,
+        http2: false,
+        net: false,
+        tls: false,
+        zlib: false,
+        child_process: false
+      }
     },
     module: {
       rules: [
